default/source/Main.hx

```diff
@@ -0,0 +1,14 @@
+package;
+
+import flixel.FlxGame;
+import openfl.Lib;
+import openfl.display.Sprite;
+
+class Main extends Sprite
+{
+    public function new()
+    {
+        super();
```

Comment:
Possibly no... I just changed the file extension, and wrote `0, 0,`.
Space were just there.